src/main/java/com/spartronics4915/frc2019/subsystems/Drive.java

```diff
@@ -181,6 +181,13 @@ private static double radiansPerSecondToTicksPer100ms(double rad_s)
         return rad_s / (Math.PI * 2.0) * Constants.kDriveEncoderPPR / 10.0;
     }
 
+    private static double ticksPer100msToInchesPerSecond(double t)
+    {
+        double ret = (t /  Constants.kDriveEncoderPPR) * (1/10) * (Constants.kDriveWheelDiameterInches * Math.PI);
```

Comment:
wouldn't hurt to add comment to motivate this expression:
should we multiply by 10 here to get from ticks/100ms to ticks/s?
another idea: decompose into more atomic computations:  ticksToInch, rotationsToInch.
now:
double ticksPerSec = t * 10;
etc..